src/Apps/Ingress/Components/HomePage/HomePage.types.ts

```diff
@@ -0,0 +1,26 @@
+import { IStyle, IStyleFunctionOrObject } from '@fluentui/react';
+import IngressAdapter from '../../../../Adapters/IngressAdapter';
+import { IExtendedTheme } from '../../../../Theming/Theme.types';
+
+export interface IHomePageProps {
+    adapter: IngressAdapter;
```

Comment:
```suggestion
    adapter: IngressAdapter | MockAdapter;
```

Reply:
@msnyder-msft mentioned why we didn't need this type. I don't recall what his reason was, but it made sense when I read it.